stdlib/3.4/asyncio/events.pyi

```diff
@@ -71,39 +80,52 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @abstractmethod
     def call_soon_threadsafe(self, callback: Callable[..., Any], *args: Any) -> Handle: ...
     @abstractmethod
+    @coroutine
     def run_in_executor(self, executor: Any,
         callback: Callable[..., Any], *args: Any) -> Future[Any]: ...
```

Comment:
If it's a coroutine it should not return a Future I think. All other coroutines are declared to return a Generator (the coroutine decorator then extracts something from that). Same for several more below.